src/Raven.Server/Documents/ETL/Providers/SQL/RelationalWriters/RelationalDatabaseWriter.cs

```diff
@@ -47,37 +47,59 @@ public RelationalDatabaseWriter(SqlEtl etl, DocumentDatabase database)
             var connectionString = etl.Configuration.Connection.ConnectionString;
             _connection.ConnectionString = connectionString;
 
-            try
-            {
-                _connection.Open();
-            }
-            catch (Exception e)
+            OpenConnection(database);
+
+            _tx = _connection.BeginTransaction();
+
+            _stringParserList = GenerateStringParsers();
+        }
+
+        private void OpenConnection(DocumentDatabase database)
+        {
+            const int maxRetries = 5;
+            var retries = 0;
+
+            while (true)
             {
-                using (_connection)
+                try
                 {
-                    database.NotificationCenter.Add(AlertRaised.Create(
-                        database.Name,
-                        SqlEtl.SqlEtlTag,
-                        $"SQL ETL could not open connection to {connectionString}",
-                        AlertType.SqlEtl_ConnectionError,
-                        NotificationSeverity.Error,
-                        key: connectionString,
-                        details: new ExceptionDetails(e)));
-
-                    throw;
+                    _connection.Open();
+                    return;
                 }
-            }
+                catch (Exception e)
+                {
+                    if (++retries < maxRetries)
+                    {
+                        if (_logger.IsInfoEnabled)
+                            _logger.Info($"Failed to open connection, retrying ({retries}/{maxRetries})", e);
 
-            _tx = _connection.BeginTransaction();
+                        Thread.Sleep(50);
+                        continue;
+                    }
 
-            _stringParserList = GenerateStringParsers();
+                    using (_connection)
+                    {
+                        database.NotificationCenter.Add(AlertRaised.Create(
+                            database.Name,
+                            SqlEtl.SqlEtlTag,
+                            $"SQL ETL could not open connection to {_connection.ConnectionString}",
```

Comment:
Maybe let's put here just the name of the connection string as it was defined in RavenDB instead of the actual connection string containing login / pass

Reply:
Changed in https://github.com/ravendb/ravendb/pull/7947